genesyscloud/resource_genesyscloud_telephony_providers_edges_site_test.go

```diff
@@ -501,7 +501,7 @@ func deleteLocationWithNumber(emergencyNumber string) error {
 						return err
 					}
 					_, err = locationsAPI.DeleteLocation(*location.Id)
-					time.Sleep(8 * time.Second)
+					time.Sleep(30 * time.Second)
```

Comment:
See earlier comments

Reply:
I think this one can be left in. It's a cleanup function that rarely runs